### PR TITLE
Search movies component created and load more option done.

### PR DESCRIPTION
**Pull request by sarthaksri-71449**
A search bar is implemented where we can search for movies. And now there is a load more button in every genres and the search results page which after getting clicked will show more movies in the page.

### CODE REVIEW

File: client/src/components/Navbar/SearchPage.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  }, [results])
+
+  const InputChange = (e) => {
+    setQuery(e.target.value);
+  }
+
+  return (
+    <div>
+        <br />
```
Comment (PatelNeelMahesh) [inline client/src/components/Navbar/SearchPage.jsx:38]: Can you explain why you need to put `<br />` tag 6 times.

Comment (sarthaksri-71449) [reply client/src/components/Navbar/SearchPage.jsx:38]: Actually, when this component was initially created then the search bar was rendered on the navbar itself which was not looking good and it was also affecting the functionality of it. So, just to bring the search bar below the navbar on the main page I inserted some breaks from the top.

Comment (PatelNeelMahesh) [reply client/src/components/Navbar/SearchPage.jsx:38]: Understood, can you change the approach in this case you can use padding as well, no need to use multiple `<br/>` tags.

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request.

**Pull request merged**